package.json

```
@@ -19,7 +19,7 @@
     "popover"
   ],
   "type": "module",
-  "main": "./dist/popover-fn.js",
+  "main": "./dist/popover.cjs.js",
```

This file doesn't exist. And I think CJS environments should be fine with the default `./dist/popover.js` build? I think I'd lean toward making the main build the default throughout, unless users specifically require the `/fn/` build. If we really need a CJS build here, we should create a new `popover.cjs.js` one (even though it's almost identical to `popover.js`).

```suggestion
  "main": "./dist/popover.js",
```